native/cocos/core/scene-graph/Node.cpp

```diff
@@ -78,6 +79,10 @@ Node::Node() : Node(EMPTY_NODE_NAME) {
 }
 
 Node::Node(const ccstd::string &name) {
+    static_assert(offsetof(Node, _padding) + sizeof(_padding) - offsetof(Node, _eventMask) == 20, "Shared memory should be 20 bytes");
+    _sharedArrayBufferObject = se::Object::createExternalArrayBufferObject(&_eventMask, 20, [](void *, size_t, void *) {});
```

Comment:
Can `_sharedArrayBufferObject` be created in `js_scene_Node_getSharedArrayBuffer`? Since it is not accessed in C++.

Reply:
`&_eventMask` is private member variable, so it can't be visited outside Node.